### PR TITLE
adds a federated search tutorial

### PR DESCRIPTION
**Pull request by sirugh**
Adds a tutorial for federated search. I had Cursor generate this tutorial from the content noted in this wiki: https://wiki.corp.adobe.com/pages/viewpage.action?pageId=3685697284

I then updated it a bit to match reality, including some screenshots, etc.

What I need:

- [x] Copy review
- [x] Content review - should we have more screenshots (outcome, etc)?
- [x] Location review - is the doc and image assets in the correct location?  @bdenham -- See the preview -- the Federated Search tutorial is under "The Boilerplate" section.  Is that the correct place?  Would it be better if it was under "Content customizations"?

### CODE REVIEW

File: src/content/docs/boilerplate/federated-search.mdx [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+- A tabbed interface displaying separate results for content and products
+- A fully functional federated search experience
+
+## Prerequisites
+
+Before starting this tutorial, make sure you have:
+
+- A working [Commerce storefront](/get-started/create-storefront/) on Edge Delivery Services
+- Basic understanding of JavaScript and asynchronous operations
+- Familiarity with [customizing blocks](/boilerplate/customizing-blocks/) in the your code repository
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
- Familiarity with [customizing blocks](/boilerplate/customizing-blocks/) in your code repository
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+- Access to your site's Helix Admin configuration (for creating the content index)
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: What is the Helix Admin configuration -- where is it --  is it config.json (maybe include the path) -- [https://](https://admin.hlx.page/config/[owner]/[repo]/[branch]/query.yaml) --  will all customers have this file -- or only if they use the config service?

File: src/content/docs/boilerplate/federated-search.mdx [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+- Adobe Commerce backend configured with product catalog
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx:41]: Can they use storefront with a third-party backend, or they have to use either Optimizer or Adobe Commerce.

File: src/content/docs/boilerplate/federated-search.mdx [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+
+## Overview
+
+This federated search approach involves:
+
+1. Creating a content index for your site's pages and content
+2. Building a custom search block that fetches both content and commerce data
+3. Implementing a tabbed UI to display results separately
+4. Handling search queries in parallel for optimal performance
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
4. Processing search queries in parallel for optimal performance
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+- Basic understanding of JavaScript and asynchronous operations
+- Familiarity with [customizing blocks](/boilerplate/customizing-blocks/) in the your code repository
+- Access to your site's Helix Admin configuration (for creating the content index)
+- Adobe Commerce backend configured with product catalog
+
+## Overview
+
+This federated search approach involves:
+
+1. Creating a content index for your site's pages and content
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
1. Creating an index for your site's pages and content
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+2. Building a custom search block that fetches both content and commerce data
+3. Implementing a tabbed UI to display results separately
+4. Handling search queries in parallel for optimal performance
+
+This approach keeps your content and commerce searches separate but presents them as a unified experience to the user.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
This method keeps content and commerce searches distinct while delivering a unified experience to the user.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 7]
```diff
@@ -0,0 +1,461 @@
...
+
+<Aside type="tip">
+**Note:** This approach is ideal when you want quick implementation, have straightforward search requirements, or want to maintain separate control over content and commerce search logic.
+</Aside>
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
<Aside type="tip" title="When to use this strategy">
This federated search strategy is ideal for scenarios where you need quick implementation, have straightforward search requirements, or want to maintain separate control over content and commerce search logic.
</Aside>
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 8]
```diff
@@ -0,0 +1,461 @@
...
+
+## Tutorial Steps
+
+### Step 1: Create the Content Index Configuration
+
+First, you need to configure a content index that will crawl and index your site's content pages. This is done by creating a `query.yaml` configuration file.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Start by setting up a content index to crawl and index your site’s pages. This is accomplished by creating a query.yaml configuration file.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 9]
```diff
@@ -0,0 +1,461 @@
...
+
+   <Aside type="note">
+   **Configuration details:**
+   - `target`: The URL path where the index JSON file will be published
+   - `include`: Glob patterns for pages to include in the index
+   - `exclude`: Paths to exclude from indexing
+   - `properties`: Metadata fields to extract from each page
+   </Aside>
+
+   After creating this configuration, the system will automatically generate a content index at `https://[your-domain]/content-index.json` whenever you publish pages.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
   After creating this configuration, the system automatically generates a content index at `https://[your-domain]/content-index.json` whenever you publish pages.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 10]
```diff
@@ -0,0 +1,461 @@
...
+
+**Expected result:** You should be able to access your content index at the configured target URL and see a JSON array of indexed pages (after you publish the pages).
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: Not sure about the "after you publish pages"  -- if i have already published pages, will I see an index for thos pages?

```suggestion
**Expected result:** After you publish pages, you can access your content index at the configured target URL and see a JSON array of indexed pages.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 11]
```diff
@@ -0,0 +1,461 @@
...
+   - `/blog/summer-sale`
+   - `/blog/holiday-shopping-guide`
+
+   Here's an example of what your content page might look like:
+
+   ![Example blog page for Winter Collection](images/federated-search-content-example.png)
+
+<Aside type="tip">
+**Content tip:** Include relevant keywords in your content that shoppers might search for. The content index contains the full text content of your pages, so descriptive content improves search results.
+</Aside>
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
<Aside type="tip" title="Content tip">
Include relevant keywords in your content that shoppers might search for. Since the content index stores the full text of your pages, well-written and relevant descriptions can significantly improve search results.
</Aside>
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 12]
```diff
@@ -0,0 +1,461 @@
...
+
+### Step 3: Create the Combined Search Block
+
+Now create a custom block that will perform federated search. This block will handle both the content and commerce searches.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Next, create a custom block to perform the federated search. This block processes both the content and commerce searches.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 13]
```diff
@@ -0,0 +1,461 @@
...
+   - `properties`: Metadata fields to extract from each page
+   </Aside>
+
+   After creating this configuration, the system will automatically generate a content index at `https://[your-domain]/content-index.json` whenever you publish pages.
+
+**Expected result:** You should be able to access your content index at the configured target URL and see a JSON array of indexed pages (after you publish the pages).
+
+### Step 2: Create Sample Content Pages
+
+To test your federated search, create and publish some content pages that will be indexed. For example, create and publish blog posts in a `/blog` directory:
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
To validate your federated search setup, publish content pages that can be indexed. For example, create and publish blog posts in a `/blog` directory:
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 14]
```diff
@@ -0,0 +1,461 @@
...
+
+Now create a custom block that will perform federated search. This block will handle both the content and commerce searches.
+
+Create a new directory: `blocks/combined-search/`
+
+Create `blocks/combined-search/combined-search.js`:
+
+<Aside type="note">
+**Full implementation:** The code below shows the key concepts. For a complete working example with full rendering logic, see the [reference implementation](https://gist.github.com/sirugh/6690b7ded67868f3b17d3c1b5a29dc76).
+</Aside>
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
<Aside type="note" title="Full implementation">
The code below shows the key concepts. For a complete working example with full rendering logic, see the [reference implementation](https://gist.github.com/sirugh/6690b7ded67868f3b17d3c1b5a29dc76).
</Aside>
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 15]
```diff
@@ -0,0 +1,461 @@
...
+
+**Key concepts explained:**
+
+1. **Content index caching** (`getContentIndex`): The content index is fetched once and cached in memory for subsequent searches, improving performance.
+
+2. **Client-side content search** (`searchContent`): Since the content index is a JSON file, we can filter it client-side using JavaScript's `filter()` method. This is fast and doesn't require additional API calls.
+
+3. **Product search** (`searchProducts`): Products are searched using the Commerce `search()` function from the Product Discovery dropin, which handles the GraphQL query to Adobe Commerce. The `visibility` filter ensures only searchable products are returned.
+
+4. **Parallel execution** (`performSearch`): Both searches run simultaneously using `Promise.all()`, which means the slower of the two doesn't hold up the faster one.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
4. **Parallel execution** (`performSearch`): Both searches execute in parallel using `Promise.all()`, ensuring that the slower query doesn’t delay the faster one.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 16]
```diff
@@ -0,0 +1,461 @@
...
+
+5. **Tabbed UI**: Results are organized in separate tabs (Products and Content) for a clean user experience.
+
+<Aside type="tip">
+**Customization:** You can add additional filters (like category or price range), adjust pagination with `pageSize`, add sorting options, customize the content search algorithm in `searchContent()`, or modify the UI structure to display results differently (e.g., in a single list instead of tabs).
+</Aside>
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
<Aside type="tip" title="Customize search behavior">
- Add additional filters (like category or price range)
- Adjust pagination with `pageSize1
- Add sorting options
- Customize the content search algorithm in `searchContent()`
- Modify the UI structure to display results differently (for example, in a single list instead of tabs).
</Aside>
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 17]
```diff
@@ -0,0 +1,461 @@
...
+
+### Step 4: Add Styling for the Search Block
+
+Create `blocks/combined-search/combined-search.css` and add the styles from the [reference implementation](https://gist.github.com/sirugh/6690b7ded67868f3b17d3c1b5a29dc76):
+
+### Step 5: Create the Search Results Page
+
+Create a new page in your project to host the combined search block. In your content authoring tool (e.g., DA.live, Google Docs, SharePoint), create a page at `/search`:
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Create a dedicated page in your project to host the combined search block. Using your content authoring tool (for example, DA.live, Google Docs, SharePoint), add a new page at `/search`.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 18]
```diff
@@ -0,0 +1,461 @@
...
+
+![Example search page](images/federated-search-search-example.png)
+
+This creates a page that uses your `combined-search` block.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
This addition creates a page that uses your `combined-search` block.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 19]
```diff
@@ -0,0 +1,461 @@
...
+
+### Step 6: Test Your Federated Search
+
+Now let's verify everything is working correctly:
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Now, let's verify everything is working correctly:
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 20]
```diff
@@ -0,0 +1,461 @@
...
+5. **Test edge cases:**
+   - Search for a term with no results
+   - Search with special characters
+   - Search with very short queries (1-2 characters)
+
+## Complete Example
+
+Here's a complete example of how the federated search works in practice:
+
+**User Journey:**
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx:356]:
```suggestion
**User Journey:**

```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 21]
```diff
@@ -0,0 +1,461 @@
...
+
+## Performance Considerations
+
+<Aside type="tip">
+**Optimization tips:**
+- The content index is fetched once and cached in memory
+- Both searches run in parallel using `Promise.all()` for optimal speed
+- Consider implementing debouncing for search-as-you-type functionality
+- For large content indexes (>1MB), consider pagination or lazy loading
+</Aside>
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
<Aside type="tip" title="Optimization tips">
- The content index is fetched once and cached in memory
- Both searches run in parallel using `Promise.all()` for optimal speed
- Consider implementing debouncing for search-as-you-type functionality
- For large content indexes (>1MB), consider pagination or lazy loading
</Aside>
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 22]
```diff
@@ -0,0 +1,461 @@
...
+
+![Example search page](images/federated-search-search-example.png)
+
+This creates a page that uses your `combined-search` block.
+
+### Step 6: Test Your Federated Search
+
+Now let's verify everything is working correctly:
+
+   1. **Test the content index:**
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: Align numbered list to left.

File: src/content/docs/boilerplate/federated-search.mdx [hunk 23]
```diff
@@ -0,0 +1,461 @@
...
+**User Journey:**
+1. Shopper types "winter" in the search box
+2. System performs parallel searches:
+   - Content search finds blog posts about "Winter Collection 2024"
+   - Product search finds products tagged with "winter"
+3. Results display in separate tabs:
+   - **Products Tab**: Displays winter jackets, boots, accessories
+   - **Content Tab**: Displays blog posts, buying guides, seasonal content
+
+**Technical Flow:**
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: Do you need to include this Technical Flow section -- seems repetitive.

```suggestion
**Technical Flow:**

```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 24]
```diff
@@ -0,0 +1,461 @@
...
+
+## Troubleshooting
+
+### Common Issues
+
+**Issue:** Content index returns 404 error
+**Solution:** Verify that you've correctly configured the `query.yaml` file and that you've published pages that match your include patterns. The index is only generated after the configuration is in place and pages are published.
+
+**Issue:** No products appear in search results
+**Solution:** Check that your GraphQL endpoint is correctly configured. Verify in browser console that the GraphQL query is executing without errors. Ensure your Adobe Commerce instance has products with the searched terms.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
**Solution:** Check that your GraphQL endpoint is correctly configured. In the browser console, verify that the GraphQL query is executing without errors. Ensure your Adobe Commerce instance has products with the searched terms.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 25]
```diff
@@ -0,0 +1,461 @@
...
+- Ensure you're caching the content index properly (see `getContentIndex()` function)
+- Consider using a CDN to cache the content-index.json file
+
+### Getting Help
+
+If you encounter issues not covered here:
+
+- Check the [FAQ section](/troubleshooting/faq/) for common storefront issues
+- Review the [Blocks Reference](/boilerplate/blocks-reference/) for block development patterns
+- Consult the [Adobe Commerce GraphQL documentation](https://developer.adobe.com/commerce/webapi/graphql/) for query troubleshooting
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: What if they are using Adobe Commerce Optimizer -- they would need to consult the Merchandising Developer Guide --  https://developer-stage.adobe.com/commerce/services/optimizer/merchandising-services/

File: src/content/docs/boilerplate/federated-search.mdx [hunk 26]
```diff
@@ -0,0 +1,461 @@
...
+
+## Alternative Approaches
+
+### Single-View Display
+
+Instead of tabs, you could display content and products in a single view with section headers:
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Instead of tabs, you can display content and products in a single view with section headers:
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 27]
```diff
@@ -0,0 +1,461 @@
...
+      <h2>Content & Resources</h2>
+      <div class="content-results-list">
+        ${renderContentItems(contentResults)}
+      </div>
+    </section>
+  `;
+}
+```
+
+This provides a single scrolling experience but may be less organized for large result sets.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
This method provides a single scrolling experience but may be less organized for large result sets.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 28]
```diff
@@ -0,0 +1,461 @@
...
+
+- A working [Commerce storefront](/get-started/create-storefront/) on Edge Delivery Services
+- Basic understanding of JavaScript and asynchronous operations
+- Familiarity with [customizing blocks](/boilerplate/customizing-blocks/) in the your code repository
+- Access to your site's Helix Admin configuration (for creating the content index)
+- Adobe Commerce backend configured with product catalog
+
+## Overview
+
+This federated search approach involves:
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Developing this federated search solution involves:
```

File: src/content/docs/boilerplate/federated-search.mdx (hunk 2 above)
Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx]: Maybe this needs rewording. There are two ways to manage the query index.

1. You create the file in your content destination (see https://www.aem.live/developer/indexing)
2. You use config service and POST to a query.yaml.

The second is what I have experience with, and what I used to create this tutorial, thus the access to config service is required for the tutorial. But we could reword it to suggest that you can also just add the index to a content index sheet (if you follow the old instructions)

File: src/content/docs/boilerplate/federated-search.mdx (hunk 3 above)
Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx:41]: Technically they could, but why would we document that :)

Also, the tutorial references the product discovery dropin, which requires ACO or AC.

File: src/content/docs/boilerplate/federated-search.mdx (hunk 10 above)
Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx]: Assuming the index definition exists in your query.yaml, then the NEXT time you publish a content doc, if it matches the index `include` rule, it will be added to the index.

I'm not sure what happens if you are creating a fresh index definition, and have not published anything, whether it will exist or not. I assume not.

File: src/content/docs/boilerplate/federated-search.mdx (hunk 25 above)
Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx]: Do you have a non-stage url?

Comment (meker12) [reply src/content/docs/boilerplate/federated-search.mdx]: https://developer.adobe.com/commerce/services/optimizer/merchandising-services/

File: src/content/docs/boilerplate/federated-search.mdx [hunk 29]
```diff
@@ -0,0 +1,392 @@
...
+- Add additional filters (like category or price range)
+- Adjust pagination with `pageSize1
+- Add sorting options
+- Customize the content search algorithm in `searchContent()`
+- Modify the UI structure to display results differently (for example, in a single list instead of tabs).
+</Aside>
+
+### Step 4: Add Styling for the Search Block
+
+Create `blocks/combined-search/combined-search.css` and add the styles from the [reference implementation](https://gist.github.com/sirugh/6690b7ded67868f3b17d3c1b5a29dc76):
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
Create `blocks/combined-search/combined-search.css` and add the styles from the [reference implementation](https://gist.github.com/sirugh/6690b7ded67868f3b17d3c1b5a29dc76).
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 30]
```diff
@@ -0,0 +1,392 @@
...
+           value: |
+             textContent(el)
+   ```
+
+   <Aside type="note">
+   **Configuration details:**
+   - `target`: The URL path where the index JSON file will be published
+   - `include`: Glob patterns for pages to include in the index
+   - `exclude`: Paths to exclude from indexing
+   - `properties`: Metadata fields to extract from each page. See [Indexing Reference](/merchants/content-customizations/indexing-reference/) for more details.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: broken link --not sure where it is supposed to go.

File: src/content/docs/boilerplate/federated-search.mdx [hunk 31]
```diff
@@ -0,0 +1,392 @@
...
+
+3. **Product search** (`searchProducts`): Products are searched using the Commerce `search()` function from the Product Discovery dropin, which handles the GraphQL query to Adobe Commerce. The `visibility` filter ensures only searchable products are returned.
+
+4. **Parallel execution** (`performSearch`): Both searches execute in parallel using `Promise.all()`, ensuring that the slower query doesn’t delay the faster one.
+
+5. **Tabbed UI**: Results are organized in separate tabs (Products and Content) for a clean user experience.
+
+<Aside type="tip" title="Customize search behavior">
+- Add additional filters (like category or price range)
+- Adjust pagination with `pageSize1
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
- Adjust pagination with `pageSize`
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 32]
```diff
@@ -0,0 +1,392 @@
...
+
+Start by setting up a content index to crawl and index your site’s pages. This is accomplished by creating a query.yaml configuration file.
+
+   Create the configuration by making a POST request to your site's admin config endpoint:
+
+   ```bash
+   POST https://admin.hlx.page/config/[owner]/sites/[repo]/content/query.yaml
+   ```
+
+   Replace `[owner]` and `[repo]`with your site's GitHub details.
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]:
```suggestion
   Replace `[owner]` and `[repo]` with your site's GitHub details.
```

File: src/content/docs/boilerplate/federated-search.mdx [hunk 33]
```diff
@@ -0,0 +1,392 @@
...
+export default async function decorate(block) {
+  // Get search query from URL
+  const urlParams = new URLSearchParams(window.location.search);
+  const query = urlParams.get('q') || '';
+
+  // Build the UI with tabs for Products and Content
+  block.innerHTML = `
+    <div class="combined-search-container">
+      <div class="search-input-wrapper">
+        <input type="text" class="search-input" value="${query}" />
```
Comment (meker12) [inline src/content/docs/boilerplate/federated-search.mdx]: cursor.ai is flagging this code sample:  <input type="text" class="search-input" value="${query}" />

The query parameter from URL should be HTML-escaped to prevent XSS. Suggest adding a note or using encodeURIComponent() or an escape function.

File: src/content/docs/boilerplate/federated-search.mdx (hunk 2 above)
Comment (meker12) [reply src/content/docs/boilerplate/federated-search.mdx]: I am not sure how to improve this -  Would making sure that you have [configured your site correctly for Enrichment](https://experienceleague.adobe.com/developer/commerce/storefront/merchants/content-customizations/enrichment/#prerequisites) address this?

Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx]: Yes! Enrichment is one index, this is just another. They would need to go in the same file.

File: src/content/docs/boilerplate/federated-search.mdx (hunk 33 above)
Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx]: I'll update.

File: src/content/docs/boilerplate/federated-search.mdx (hunk 30 above)
Comment (sirugh) [reply src/content/docs/boilerplate/federated-search.mdx]: Should be here: https://www.aem.live/developer/indexing I'll update.

**Comment by bdenham** [comment]
I added placeholders in the docs with the following image names. They should go in this folder: src/content/docs/boilerplate/images/

federated-search-content-index.png - the JSON output
federated-search-results.png - full search page with results
federated-search-products-tab.png - products tab view (optional but helpful)
federated-search-content-tab.png - content tab view  (optional but helpful)

**Review by bdenham — APPROVED** [review]
